### ISSUE TITLE
More inclusive language in the introduction docs

### ISSUE THREAD

**Issue by carolstran** [cypress-io/cypress-documentation#2120]
## I'm submitting a...

```
[ ] Bug report
[x] Content update
[ ] Process update (build, deployment, ... )
```

## Type of bug / changes

:wave: I noticed that there are many instances of the word "simple" in your documentation. Unfortunately, words like this can make people feel frustrated and isolated while reading (myself included, hence the issue) - especially if they are facing issues.

If it's alright with your team, I'd like to take a shot at updating the content to make it more inclusive. Particularly the [Cypress Is Simple](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Cypress-Is-Simple) section of the Introduction, but perhaps some other places where it pops up as well. I enjoy using Cypress so I'd be happy to help 😊 

Also if you're interested or want more information on this topic, I'd highly recommend watching Jim Fisher's [Don't Say Simply](https://www.youtube.com/watch?v=gsT2BBWBVmM) talk from Write the Docs Prague 2018 ✍️ 🤓 

Thank you for the work you do and please let me know if you need more details!

**Comment by sbinlondon** [comment]
Just wanted to add a +1 to @carolstran's issue. I used Cypress for the first time in an open source repo this week. While I love the documentation on the whole and think it's been one of the more thorough and well-explained bits of software I've used, I did chuckle a bit at the 'Cypress is simple'. I also ran into some bits of frustration that any person using something completely new for the first time might run into, and it didn't feel 'simple' then. Integration testing can be a hard concept. You've got a great product and great documentation - I think this change will definitely help make it even better.

**Comment by flotwig** [comment]
@carolstran You're right, it may even be more accurate to update that title from "_Cypress Is Simple_" to "_Cypress Can Be Simple (Sometimes)_" :) 

We definitely don't want to frustrate anyone by implying that issues they're having are trivial. We'd be happy to merge a PR that updates the docs to make them more inclusive.

**Comment by carolstran** [comment]
Awesome! I'll get to work, thank you @flotwig 🙌

**Comment by carolstran** [comment]
Not sure if I should keep the issue open or close it so I'll leave it to you 😄

**Comment by flotwig** [comment]
@carolstran When you open your PR, put this text in the beginning:

> Closes #2120

GitHub will link it to this issue and automatically close this when your PR is merged.
